toml_map.go

```diff
@@ -0,0 +1,69 @@
+package altsrc
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type tomlMap struct {
+	Map map[any]any
+}
+
+func (tm *tomlMap) UnmarshalTOML(i any) error {
+	if v, err := unmarshalMap(i); err == nil {
+		tm.Map = v
+	} else {
+		return err
+	}
+
+	return nil
+}
+
+func unmarshalMap(i any) (ret map[any]any, err error) {
+	ret = make(map[any]any)
```

Comment:
map[string]any  ?

Reply:
idk; this was a _move_ too 🤔  I was thinking I'd avoid changing the implementation at this time but I can!